Libraries/dotNetRDF/Writing/Formatting/BaseFormatter.cs

```diff
@@ -130,7 +130,7 @@ public virtual String FormatUri(String u)
         /// <returns></returns>
         public virtual String FormatUri(Uri u)
         {
-            return FormatUri(u.AbsoluteUri);
+            return FormatUri(u.ToString());
```

Comment:
This is where Unicode character were unnecessarily escaped because `AbsoluteUri` is always percent-encoded